plugins/module_utils/foreman_helper.py

```diff
@@ -87,6 +87,32 @@ def ping(self):
 
 class ForemanApypieAnsibleModule(ForemanBaseAnsibleModule):
 
+    def _patch_location_api(self):
+        """This is a workaround for the broken taxonomies apidoc in foreman."""
+
+        _location_organiuzations_parameter = {
+            u'validations': [],
```

Comment:
not sure we need to mark all these *explicitly* as unicode

Reply:
I'd say, it does not hurt. (Python loves _explicit_.)